### PR TITLE
added devnet to not require latest major

### PR DESCRIPTION
**Pull request by jmatosp**

### CODE REVIEW

File: client/cmd/kcoin/main.go [hunk 1]
```diff
@@ -257,8 +257,8 @@ func startNode(ctx *cli.Context, stack *node.Node) {
 	debug.Memsize.Add("node", stack)
 	setupLogging(ctx)
 
-	// make use client runs latest major version if not in testnet mode, need to check pre Start node!
-	if !ctx.GlobalBool(utils.TestnetFlag.Name) {
+	// make use client runs latest major version if not in testnet mode or devnet, need to check pre Start node!
+	if !isTestnetOrDevnet(ctx) {
```
Comment (JekaMas) [inline client/cmd/kcoin/main.go]: You don't use `isTestnetOrDevnet` anywhere else, maybe it makes sense to use straightforward condition `!isTestnet(ctx) && !isDevnet(ctx)` ?

Comment (jmatosp) [reply client/cmd/kcoin/main.go]: just for readability, easy to mix up those Nots and &&

Comment (jmatosp) [reply client/cmd/kcoin/main.go]: or `if !(isTestnet(ctx) || isDevnet(ctx))`? :)

Comment (JekaMas) [reply client/cmd/kcoin/main.go]: ok, for me it's easier to read logic expressions as it is, ie `!isTestnet(ctx) && !isDevnet(ctx)`.

**Pull request merged**